cypress/e2e/spec.ts

```diff
@@ -138,27 +142,27 @@ describe('workspace-project App', () => {
     });
     cy.contains('Dummy Workflow 1').should('be.visible', { timeout: 10000 });
     cy.contains('settings');
-    cy.get('mat-toolbar > button mat-icon').contains('settings').click();
+    cy.get('mat-toolbar > button mat-icon').contains('settings').click({force: true});
 
-    cy.get('app-workflow-sidenav').contains('Add Task').click();
+    cy.get('app-workflow-sidenav').contains('Add Task').click({force: true});
     cy.contains('Select Task');
-    cy.get('mat-dialog-container').contains('Mapping').click();
-    cy.get('app-add-block-dialog .mat-dialog-actions').contains('Add Task').click();
+    cy.get('mat-dialog-container').contains('Mapping').click({force: false});
+    cy.get('app-workflow-sidenav').contains('Add Task').click({force: true});
 
-    cy.get('app-root mat-toolbar').contains('menu').click();
+    cy.get('app-root mat-toolbar').contains('menu').click({force: true});
     // click in the menu to navigate away
-    cy.contains('Dashboard').click();
+    cy.contains('Dashboard').click({force: true});
     // confirm that we're not on the dashboard
     cy.location().should((loc) => {
       expect(loc.pathname).to.eq('/TESTING/dummy1');
     });
     // try again
-    cy.contains('Dashboard').click();
+    cy.contains('Dashboard').click({force: true});
     // this time we got there
     cy.location().should((loc) => {
       expect(loc.pathname).to.eq('/dashboard');
     });
   })
-
+  */
```

Comment:
Need an issue for this too.

Reply:
Now there is an issue for it: https://github.com/kendraio/kendraio-app/issues/428

Reply:
Add the issue to the code comment?